Description: Fix some typos.
Forwarded: not yet
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2015-11-19

--- a/src/options.c
+++ b/src/options.c
@@ -2488,7 +2488,7 @@
         section->option.sessiond=1;
 #ifdef SSL_OP_NO_TICKET
         /* disable RFC4507 support introduced in OpenSSL 0.9.8f */
-        /* this prevents session callbacks from beeing executed */
+        /* this prevents session callbacks from being executed */
         section->ssl_options_set|=SSL_OP_NO_TICKET;
 #endif
         if(!name2addr(&section->sessiond_addr, arg, 0))
--- a/src/ssl.c
+++ b/src/ssl.c
@@ -38,7 +38,7 @@
 #include "common.h"
 #include "prototypes.h"
 
-    /* global OpenSSL initalization: compression, engine, entropy */
+    /* global OpenSSL initialization: compression, engine, entropy */
 NOEXPORT void cb_free(void *parent, void *ptr, CRYPTO_EX_DATA *ad,
     int idx, long argl, void *argp);
 #ifndef OPENSSL_NO_COMP
