Description: Fix some typos.
Forwarded: not yet
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2015-06-13

--- a/doc/en/VNC_StunnelHOWTO.html
+++ b/doc/en/VNC_StunnelHOWTO.html
@@ -36,8 +36,8 @@
 <P STYLE="margin-bottom: 0cm"><BR>
 </P>
 <OL>
-	<LI><P STYLE="margin-bottom: 0cm">Download and install openSSL,
-	SSLEay, and Stunnel on the Linux/Unix box. Download the modules.</P>
+	<LI><P STYLE="margin-bottom: 0cm">Download and install OpenSSL,
+	SSLeay, and Stunnel on the Linux/Unix box. Download the modules.</P>
 </OL>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">a)
 [root@anthrax$]gunzip openssl-x.xx.tar.gz (repeat for all 3 the
@@ -52,7 +52,7 @@
 	save the file as VNCRegEdit.REG on the Windows 2000 box</P>
 </OL>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">--cut here and copy
-to VNCRegEdit.REG the double click file to
+to VNCRegEdit.REG then double click the file to
 import--<BR>REGEDIT4<BR><BR>[HKEY_LOCAL_MACHINE\Software\ORL\WinVNC3]<BR>AllowLoopback=dword:00000001<BR><BR>[HKEY_LOCAL_MACHINE\Software\ORL\WinVNC3\Default]<BR>AllowLoopback=dword:00000001<BR>--stop
 here--<BR><BR>
 </P>
@@ -87,7 +87,7 @@
 	execute the following command and let it run in its own terminal.</P>
 </OL>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">stunnel -d 5900 -r
-unix.ip.adress:5900 -c</P>
+unix.ip.address:5900 -c</P>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">.</P>
 <OL>
 	<LI><P STYLE="margin-bottom: 0cm">And on the Windows 2000 machine
@@ -109,7 +109,7 @@
 2000 command as follows: 
 </P>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">stunnel -d 5902 -r
-unix.ip.adress:5902</P>
+unix.ip.address:5902</P>
 <P STYLE="margin-left: 0.5cm; margin-bottom: 0cm">and remember to
 start another vncserver on the Linux box for each VNC display</P>
 <P STYLE="margin-bottom: 0cm"><BR>
@@ -165,11 +165,11 @@
 <P STYLE="margin-bottom: 0cm"><BR>
 </P>
 <P STYLE="margin-bottom: 0cm">To connect from the client machine you
-need to enter the client machines IP address and the &quot;display&quot;
+need to enter the client machine's IP address and the &quot;display&quot;
 (from the port conversion). But VNC will think that you are trying to
 connect to the local machine and does not allow this. To override
-this add the following to you registry.<BR><BR>--cut here and copy to
-anything.reg. the double click file to
+this add the following to your registry.<BR><BR>--cut here and copy to
+anything.reg. then double click the file to
 import--<BR>REGEDIT4<BR><BR>[HKEY_LOCAL_MACHINE\Software\ORL\WinVNC3]<BR>AllowLoopback=dword:00000001<BR><BR>[HKEY_LOCAL_MACHINE\Software\ORL\WinVNC3\Default]<BR>AllowLoopback=dword:00000001<BR>--stop
 here--<BR><BR>Now VNC will not complain. So you need to always run
 stunnel in client mode on the Windows machine and then connect with
@@ -182,9 +182,9 @@
 <P STYLE="margin-bottom: 0cm"><BR>
 </P>
 <P STYLE="margin-bottom: 0cm">Unfortunately this will not work well
-with the build in web version. If you did not known about it, try
+with the built-in web version. If you did not known about it, try
 http'ing into a machine running VNC server on it, to port 58XX (where
 XX is the display number), and the Java client will be loaded.<BR><BR>
 </P>
 </BODY>
-</HTML>
\ No newline at end of file
+</HTML>
--- a/doc/pl/tworzenie_certyfikatow.html
+++ b/doc/pl/tworzenie_certyfikatow.html
@@ -93,7 +93,7 @@
 # private random number file</I>
 <BR><I>&nbsp;</I>
 <BR><I>x509_extensions = usr_cert&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-# The extentions to add to the cert</I>
+# The extensions to add to the cert</I>
 <BR><I>crl_extensions&nbsp; = crl_ext&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 # Extensions to add to CRL</I>
 <BR><I>default_days&nbsp;&nbsp;&nbsp; = 365&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
@@ -147,7 +147,7 @@
 <BR><I>distinguished_name&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; = req_distinguished_name</I>
 <BR><I>attributes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 = req_attributes</I>
-<BR><I>x509_extensions = v3_ca # The extentions to add to the self signed
+<BR><I>x509_extensions = v3_ca # The extensions to add to the self signed
 cert</I>
 <BR><I>&nbsp;</I>
 <BR><I>[ req_distinguished_name ]</I>
--- a/tools/ca.pl
+++ b/tools/ca.pl
@@ -61,5 +61,5 @@
 
 sub Error {
     print "Content-type: text/html\n\n";
-    print "<P><P><center><H1>Cant open file</H1></center>\n";
+    print "<P><P><center><H1>Can't open file</H1></center>\n";
 }
--- a/doc/stunnel.pod
+++ b/doc/stunnel.pod
@@ -85,11 +85,11 @@
 
 =item B<-reload> (Windows NT and later only)
 
-Reload configuration file of the running NT Service
+Reload the configuration file of the running NT Service
 
 =item B<-reopen> (Windows NT and later only)
 
-Reopen log file of the running NT Service
+Reopen the log file of the running NT Service
 
 =item B<-exit> (Win32 only)
 
@@ -152,11 +152,11 @@
 
 directory to chroot B<stunnel> process
 
-B<chroot> keeps B<stunnel> in chrooted jail.  I<CApath>, I<CRLpath>, I<pid>
+B<chroot> keeps B<stunnel> in a chrooted jail.  I<CApath>, I<CRLpath>, I<pid>
 and I<exec> are located inside the jail and the patches have to be relative
 to the directory specified with B<chroot>.
 
-Several functions of the operating system also need their files to be located within chroot jail, e.g.:
+Several functions of the operating system also need their files to be located within the chroot jail, e.g.:
 
 =over 4
 
@@ -191,7 +191,7 @@
 
 debugging level
 
-Level is a one of the syslog level names or numbers
+Level is one of the syslog level names or numbers
 emerg (0), alert (1), crit (2), err (3), warning (4), notice (5),
 info (6), or debug (7).  All logs for the specified level and
 all levels numerically less than it will be shown.  Use I<debug = debug> or
@@ -206,7 +206,7 @@
 
 path to Entropy Gathering Daemon socket
 
-Entropy Gathering Daemon socket to use to feed B<OpenSSL> random number
+Entropy Gathering Daemon socket to use to feed the B<OpenSSL> random number
 generator.  (Available only if compiled with B<OpenSSL 0.9.5a> or higher)
 
 =item B<engine> = auto | ENGINE_ID
@@ -215,7 +215,7 @@
 
 default: software-only cryptography
 
-Here is an example of advanced engine configuration to read private key from an
+Here is an example of advanced engine configuration to read the private key from an
 OpenSC engine
 
     engine=dynamic
@@ -251,7 +251,7 @@
 
 Enable or disable FIPS 140-2 mode.
 
-This option allows to disable entering FIPS mode if B<stunnel> was compiled
+This option allows you to disable entering FIPS mode if B<stunnel> was compiled
 with FIPS 140-2 support.
 
 default: no (since version 5.00)
@@ -290,7 +290,7 @@
 
 log file handling
 
-This option allows to choose whether the log file (specified with the I<output>
+This option allows you to choose whether the log file (specified with the I<output>
 option) is appended or overwritten when opened or re-opened.
 
 default: append
@@ -308,7 +308,7 @@
 
 If the argument is empty, then no pid file will be created.
 
-I<pid> path is relative to I<chroot> directory if specified.
+I<pid> path is relative to the I<chroot> directory if specified.
 
 =item B<RNDbytes> = BYTES
 
@@ -352,10 +352,10 @@
 
 =item B<socket> = a|l|r:OPTION=VALUE[:VALUE]
 
-Set an option on accept/local/remote socket
+Set an option on the accept/local/remote socket
 
-The values for linger option are l_onof:l_linger.
-The values for time are tv_sec:tv_usec.
+The values for the linger option are l_onof:l_linger.
+The values for the time are tv_sec:tv_usec.
 
 Examples:
 
@@ -386,7 +386,7 @@
 
 =head2 SERVICE-LEVEL OPTIONS
 
-Each configuration section begins with service name in square brackets.
+Each configuration section begins with a service name in square brackets.
 The service name is used for libwrap (TCP Wrappers) access control and lets
 you distinguish B<stunnel> services in your log files.
 
@@ -413,20 +413,20 @@
 Certificate Authority directory
 
 This is the directory in which B<stunnel> will look for certificates when using
-the I<verify>.  Note that the certificates in this directory should be named
+the I<verify> option.  Note that the certificates in this directory should be named
 XXXXXXXX.0 where XXXXXXXX is the hash value of the DER encoded subject of the
 cert.
 
 The hash algorithm has been changed in B<OpenSSL 1.0.0>.  It is required to
 c_rehash the directory on upgrade from B<OpenSSL 0.x.x> to B<OpenSSL 1.x.x>.
 
-I<CApath> path is relative to I<chroot> directory if specified.
+I<CApath> path is relative to the I<chroot> directory if specified.
 
 =item B<CAfile> = CERT_FILE
 
 Certificate Authority file
 
-This file contains multiple CA certificates, used with the I<verify>.
+This file contains multiple CA certificates, used with the I<verify> option.
 
 =item B<cert> = PEM_FILE
 
@@ -468,7 +468,7 @@
 
 Select permitted SSL ciphers
 
-A colon delimited list of the ciphers to allow in the SSL connection,
+A colon-delimited list of the ciphers to allow in the SSL connection,
 for example DES-CBC3-SHA:IDEA-CBC-MD5.
 
 =item B<client> = yes | no
@@ -494,25 +494,25 @@
 Certificate Revocation Lists directory
 
 This is the directory in which B<stunnel> will look for CRLs when
-using the I<verify>. Note that the CRLs in this directory should
+using the I<verify> option. Note that the CRLs in this directory should
 be named XXXXXXXX.r0 where XXXXXXXX is the hash value of the CRL.
 
 The hash algorithm has been changed in B<OpenSSL 1.0.0>.  It is required to
 c_rehash the directory on upgrade from B<OpenSSL 0.x.x> to B<OpenSSL 1.x.x>.
 
-I<CRLpath> path is relative to I<chroot> directory if specified.
+I<CRLpath> path is relative to the I<chroot> directory if specified.
 
 =item B<CRLfile> = CERT_FILE
 
 Certificate Revocation Lists file
 
-This file contains multiple CRLs, used with the I<verify>.
+This file contains multiple CRLs, used with the I<verify> option.
 
 =item B<curve> = NID
 
 specify ECDH curve name
 
-To get a list of supported cuves use:
+To get a list of supported curves use:
 
     openssl ecparam -list_curves
 
@@ -522,7 +522,7 @@
 
 connection identifier type
 
-This identifier allows to distinguish log entries generated for each of the
+This identifier allows you to distinguish log entries generated for each of the
 connections.
 
 Currently supported types:
@@ -561,7 +561,7 @@
 
 =item B<delay> = yes | no
 
-delay DNS lookup for I<connect> option
+delay DNS lookup for the I<connect> option
 
 This option is useful for dynamic DNS, or when DNS is not available during
 B<stunnel> startup (road warrior VPN, dial-up configurations).
@@ -585,19 +585,19 @@
 
 =item B<exec> = EXECUTABLE_PATH
 
-execute local inetd-type program 
+execute a local inetd-type program 
 
-I<exec> path is relative to I<chroot> directory if specified.
+I<exec> path is relative to the I<chroot> directory if specified.
 
-The following environmental variables are set on Unix platform:
+The following environmental variables are set on Unix platforms:
 REMOTE_HOST, REMOTE_PORT, SSL_CLIENT_DN, SSL_CLIENT_I_DN.
 
 =item B<execArgs> = $0 $1 $2 ...
 
-arguments for I<exec> including program name ($0)
+arguments for I<exec> including the program name ($0)
 
 Quoting is currently not supported.
-Arguments are separated with arbitrary number of whitespaces.
+Arguments are separated with an arbitrary amount of whitespace.
 
 =item B<failover> = rr | prio
 
@@ -614,15 +614,15 @@
 
 =item B<key> = KEY_FILE
 
-private key for certificate specified with I<cert> option
+private key for the certificate specified with I<cert> option
 
-Private key is needed to authenticate certificate owner.
+A private key is needed to authenticate the certificate owner.
 Since this file should be kept secret it should only be readable
-to its owner.  On Unix systems you can use the following command:
+by its owner.  On Unix systems you can use the following command:
 
     chmod 600 keyfile
 
-default: value of I<cert> option
+default: the value of the I<cert> option
 
 =item B<libwrap> = yes | no
 
@@ -632,8 +632,8 @@
 
 =item B<local> = HOST
 
-IP of the outgoing interface is used as source for remote connections.
-Use this option to bind a static local IP address, instead.
+By default, the IP address of the outgoing interface is used as the source for remote connections.
+Use this option to bind a static local IP address instead.
 
 =item B<sni> = SERVICE:SERVER_PATTERN (server mode)
 
@@ -641,29 +641,29 @@
 Name Indication TLS extension (RFC 3546).
 
 I<service_name> specifies the master service that accepts client connections
-with I<accept> option.  I<server_name_pattern> specifies the host name to be
-redirected.  The pattern may start with '*' character, e.g. '*.example.com'.
+with the I<accept> option.  I<server_name_pattern> specifies the host name to be
+redirected.  The pattern may start with the '*' character, e.g. '*.example.com'.
 Multiple slave services are normally specified for a single master service.
-I<sni> option can also be specified more than once within a single slave
+The I<sni> option can also be specified more than once within a single slave
 service.
 
 This service, as well as the master service, may not be configured in client
 mode.
 
-I<connect> option of the slave service is ignored when I<protocol> option is
-specified, as I<protocol> connects remote host before TLS handshake.
+The I<connect> option of the slave service is ignored when the I<protocol> option is
+specified, as I<protocol> connects to the remote host before TLS handshake.
 
-Libwrap checks (Unix only) are performed twice: with master service name after
-TCP connection is accepted, and with slave service name during TLS handshake.
+Libwrap checks (Unix only) are performed twice: with the master service name after
+TCP connection is accepted, and with the slave service name during the TLS handshake.
 
-Option I<sni> is only available when compiled with B<OpenSSL 1.0.0> and later.
+The I<sni> option is only available when compiled with B<OpenSSL 1.0.0> and later.
 
 =item B<sni> = SERVER (client mode)
 
 Use the parameter as the value of TLS Server Name Indication (RFC 3546)
 extension.
 
-Option I<sni> is only available when compiled with B<OpenSSL 1.0.0> and later.
+The I<sni> option is only available when compiled with B<OpenSSL 1.0.0> and later.
 
 =item B<OCSP> = URL
 
@@ -715,7 +715,7 @@
 
 This option enables initial, protocol-specific negotiation of the SSL/TLS
 encryption.
-I<protocol> option should not be used with SSL encryption on a separate port.
+The I<protocol> option should not be used with SSL encryption on a separate port.
 
 Currently supported protocols:
 
@@ -782,7 +782,7 @@
 
 authentication type for protocol negotiations
 
-Currently authentication type only applies to the 'connect' protocol.
+Currently the authentication type only applies to the 'connect' protocol.
 
 default: basic
 
@@ -790,11 +790,11 @@
 
 destination address for protocol negotiations
 
-I<protocolHost> specifies the final SSL server to be connected by the proxy,
+I<protocolHost> specifies the final SSL server to be connected to by the proxy,
 and not the proxy server directly connected by B<stunnel>.
 The proxy server should be specified with the 'connect' option.
 
-Currently protocol destination address only applies to 'connect' protocol.
+Currently the protocol destination address only applies to 'connect' protocol.
 
 =item B<protocolPassword> = PASSWORD
 
@@ -826,7 +826,7 @@
 
 =item B<pty> = yes | no (Unix only)
 
-allocate pseudoterminal for 'exec' option
+allocate a pseudoterminal for 'exec' option
 
 =item B<redirect> = [HOST:]PORT
 
@@ -854,7 +854,7 @@
 
 =item B<reset> = yes | no
 
-attempt to use TCP RST flag to indicate an error
+attempt to use the TCP RST flag to indicate an error
 
 This option is not supported on some platforms.
 
@@ -874,7 +874,7 @@
 entries.
 
 The value of 0 can be used for unlimited size.  It is not recommended
-for production use due to the risk of memory exhaustion DoS attack.
+for production use due to the risk of a memory exhaustion DoS attack.
 
 =item B<sessionCacheTimeout> = TIMEOUT
 
@@ -906,7 +906,7 @@
 
 =item B<TIMEOUTconnect> = SECONDS
 
-time to wait to connect a remote host
+time to wait to connect to a remote host
 
 =item B<TIMEOUTidle> = SECONDS
 
@@ -926,7 +926,7 @@
 
 =item I<source>
 
-Re-write address to appear as if wrapped daemon is connecting
+Re-write the address to appear as if a wrapped daemon is connecting
 from the SSL client machine instead of the machine running B<stunnel>.
 
 This option is currently available in:
@@ -936,7 +936,7 @@
 =item Remote mode (I<connect> option) on I<Linux E<gt>=2.6.28>
 
 This configuration requires B<stunnel> to be executed as root and without
-I<setuid> option.
+the I<setuid> option.
 
 This configuration requires the following setup for iptables and routing
 (possibly in /etc/rc.local or equivalent file):
@@ -949,25 +949,25 @@
     ip route add local 0.0.0.0/0 dev lo table 100
     echo 0 >/proc/sys/net/ipv4/conf/lo/rp_filter
 
-B<stunnel> must also to be executed as root and without I<setuid> option.
+B<stunnel> must also to be executed as root and without the I<setuid> option.
 
 =item Remote mode (I<connect> option) on I<Linux 2.2.x>
 
-This configuration requires kernel to be compiled with I<transparent proxy>
+This configuration requires the kernel to be compiled with the I<transparent proxy>
 option.
 Connected service must be installed on a separate host.
 Routing towards the clients has to go through the B<stunnel> box.
 
-B<stunnel> must also to be executed as root and without I<setuid> option.
+B<stunnel> must also to be executed as root and without the I<setuid> option.
 
 =item Remote mode (I<connect> option) on I<FreeBSD E<gt>=8.0>
 
 This configuration requires additional firewall and routing setup.
-B<stunnel> must also to be executed as root and without I<setuid> option.
+B<stunnel> must also to be executed as root and without the I<setuid> option.
 
 =item Local mode (I<exec> option)
 
-This configuration works by pre-loading I<libstunnel.so> shared library.
+This configuration works by pre-loading the I<libstunnel.so> shared library.
 _RLD_LIST environment variable is used on Tru64, and LD_PRELOAD variable on
 other platforms.
 
@@ -975,7 +975,7 @@
 
 =item I<destination>
 
-Original destination is used instead of I<connect> option.
+The original destination is used instead of the I<connect> option.
 
 A service section for transparent destination may look like this:
 
@@ -999,7 +999,7 @@
     /sbin/iptables -t nat -I PREROUTING -p tcp --dport <redirected_port> \
         -i eth0 -j DNAT --to-destination <local_ip>:<stunnel_port>
 
-Transparent destination option is currently only supported on Linux.
+The transparent destination option is currently only supported on Linux.
 
 =item I<both>
 
@@ -1013,40 +1013,40 @@
 
 =item I<yes>
 
-This options has been renamed to I<source>.
+This option has been renamed to I<source>.
 
 =item I<no>
 
-This options has been renamed to I<none>.
+This option has been renamed to I<none>.
 
 =back
 
 
 =item B<verify> = LEVEL
 
-verify peer certificate
+verify the peer certificate
 
 =over 4
 
 =item level 0
 
-Request and ignore peer certificate.
+Request and ignore the peer certificate.
 
 =item level 1
 
-Verify peer certificate if present.
+Verify the peer certificate if present.
 
 =item level 2
 
-Verify peer certificate.
+Verify the peer certificate.
 
 =item level 3
 
-Verify peer with locally installed certificate.
+Verify the peer with locally installed certificate.
 
 =item level 4
 
-Ignore CA chain and only verify peer certificate.
+Ignore the CA chain and only verify the peer certificate.
 
 =item default
 
@@ -1054,7 +1054,7 @@
 
 =back
 
-It is important to understand, that this option was solely designed for access
+It is important to understand that this option was solely designed for access
 control and not for authorization.  Specifically for level 2 every non-revoked
 certificate is accepted regardless of its Common Name.  For this reason a
 dedicated CA should be used with level 2, and not a generic CA commonly used
@@ -1104,16 +1104,16 @@
 
 =back
 
-The use of 'setuid' option will also prevent B<stunnel> from binding privileged
+The use of the 'setuid' option will also prevent B<stunnel> from binding to privileged
 (<1024) ports during configuration reloading.
 
-When 'chroot' option is used, B<stunnel> will look for all its files (including
-configuration file, certificates, log file and pid file) within the chroot
+When the 'chroot' option is used, B<stunnel> will look for all its files (including
+the configuration file, certificates, the log file and the pid file) within the chroot
 jail.
 
 =item SIGUSR1
 
-Close and reopen B<stunnel> log file.
+Close and reopen the B<stunnel> log file.
 This function can be used for log rotation.
 
 =item SIGTERM, SIGQUIT, SIGINT
@@ -1228,8 +1228,8 @@
 
 The order of contents of the I<.pem> file is important.  It should contain the
 unencrypted private key first, then a signed certificate (not certificate
-request).  There should be also empty lines after certificate and private key.
-Plaintext certificate information appended on the top of generated certificate
+request).  There should also be empty lines after the certificate and the private key.
+Any plaintext certificate information appended on the top of generated certificate
 should be discarded. So the file should look like this:
 
     -----BEGIN RSA PRIVATE KEY-----
@@ -1243,7 +1243,7 @@
 
 =head2 RANDOMNESS
 
-B<stunnel> needs to seed the PRNG (pseudo random number generator) in
+B<stunnel> needs to seed the PRNG (pseudo-random number generator) in
 order for SSL to use good randomness.  The following sources are loaded
 in order until sufficient random data has been gathered:
 
@@ -1329,7 +1329,7 @@
 
 =head1 BUGS
 
-Option I<execArgs> and Win32 command line does not support quoting.
+The I<execArgs> option and the Win32 command line do not support quoting.
 
 
 =head1 SEE ALSO
