Description: Remove a redundant null pointer check.
Forwarded: not yet
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2015-06-04

--- a/src/sthreads.c
+++ b/src/sthreads.c
@@ -437,8 +437,7 @@
     s_log(LOG_DEBUG, "Creating a new thread");
     if((long)_beginthread((void(*)(void *))cli, arg->opt->stack_size, arg)==-1) {
         ioerror("_beginthread");
-        if(arg)
-            str_free(arg);
+        str_free(arg);
         if(s>=0)
             closesocket(s);
         return -1;
