Description: Check for Large File Support during autoconfiguration.
Forwarded: not-yet
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2016-05-04

--- a/configure.ac
+++ b/configure.ac
@@ -175,6 +175,8 @@
 AC_MSG_RESULT([$DEFAULT_GROUP])
 AC_SUBST([DEFAULT_GROUP])
 
+AC_SYS_LARGEFILE
+
 AC_MSG_NOTICE([**************************************** header files])
 # AC_HEADER_DIRENT
 # AC_HEADER_STDC
